compete.html

```diff
@@ -130,12 +130,12 @@ <h3>Building a robot</h3>
       <a name="signup"></a><h2>Want to compete?</h2>
       <div class="column m-8-12 m-offset-2-12 text-left">
         <p>
-          Please register your interest below, we'll let you know when formal entries open later this year.
+          Please register your interest below. We'll let you know once your place is confirmed.
         </p>
         <p class="text-heavy">
           You must be over 18 to run a Student Robotics team and so you must also be over 18 to fill in this form.
         </p>
-        {% include team-signup.html %}
+        <a class="button" href="https://forms.gle/mCfqNHrF2JWRboch8" target="_blank">Sign up for Student Robotics 2020</a>
```

Comment:
Maybe `iframe` the form in?

Reply:
https://github.com/srobo/website/pull/156/commits/4036a4086520fd3c89762f10ac244b77dd831294